src/index.html

```diff
@@ -7,6 +7,498 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="nav">
+      <a
+        class="nav__link-logo"
+        href="#home"
+      >
+        <img
+          class="nav__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav__container">
+        <ul
+        class="nav__container-list"
+        >
```

Comment:
```suggestion
        <ul class="nav__container-list">
```
if you have fewer than 3 attributes in a tag there is no need to move it to  the new line